opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp

```diff
@@ -343,16 +350,50 @@ namespace Opm {
                     }
 
                     const double newTimeStep = restartFactor_ * dt;
-                    // we need to revise this
-                    substepTimer.provideTimeStepEstimate(newTimeStep);
-                    if (solverVerbose_) {
-                        std::string msg;
-                        msg = causeOfFailure + "\nTimestep chopped to "
-                            + std::to_string(unit::convert::to(substepTimer.currentStepLength(), unit::day)) + " days\n";
-                        OpmLog::problem(msg);
-                    }
+                    const double minimumChoppedTimestep = 0.25 * unit::day;
+                    if (newTimeStep > minimumChoppedTimestep) {
```

Comment:
Should the minimumChoppedTimestep a user defined parameter?

Reply:
I think so. At the point, it is claimed to be hard-coded temporarily for now.

Reply:
The minimumChoppedTimestep logic is the cause of the sudden failure in the new norne model I am currently testing. How sensitive is this hard-coded parameter for the convergence of the prediction case?

Reply:
I  think, if the well is in history matching model, it should not be shut no matter what chopping happened. Unless we change this opinion.

> How sensitive is this hard-coded parameter for the convergence of the prediction case?

It should not be sensitive. More testing might be needed to make a better one.